react/src/components/organisms/HelpTip/index.js

```diff
@@ -67,68 +73,72 @@ class HelpTip extends Component {
       'ma__help-tip__text-direct': true,
       [`ma__help-tip__text-direct--${theme}`]: theme
     });
+    const splitText = this.getSplitText(text, triggerText);
+    const childArray = children && (Array.isArray(children) ? children : [children]);
 
     return(
       <span className={baseClass} id={id}>
-        <span className="ma__help-tip__label" id={labelId}>
-          {this.buildDangerouslyIfHasMarkup(this.props.textBefore, hasMarkup)}
-          {/* can't use a button b/c disrupts fieldsets when used inside of legend */}
-          <div
-            className={`ma__help-tip__trigger ${this.state.isOpen ? 'active' : ''}`}
-            onClick={this.toggleOpen}
-            onKeyUp={this.toggleOpenForKeyUp}
-            tabIndex="0"
-            role="button"
-            aria-describedby={id}
-            aria-expanded={this.state.isOpen}
-          >
-            {this.buildDangerouslyIfHasMarkup(triggerText, hasMarkup)}
-            <Icon name="questionmark" svgHeight={15} svgWidth={15} />
-          </div>
-          {this.buildDangerouslyIfHasMarkup(textAfter, hasMarkup)}
-        </span>
-        <Collapse in={this.state.isOpen} dimension="height" className={helpTextContainer}>
-          <div className="ma__help-tip__content">
-            <div
+        {triggerText.map((trigger, index) => (
+          <span key={`HelpTip-${labelId}-${index}`} className="ma__help-tip__label" id={`${labelId}-${index}`}>
```

Comment:
```suggestion
          <span key={`HelpTip-${id}-${index}`} className="ma__help-tip__label" id={`${labelId}-${index}`}>
```
Can we remove the labelId required prop?